todo_app/data/roleChecker.py

```diff
@@ -0,0 +1,11 @@
+from flask_login import current_user
+from flask import redirect
+import os
+
+def ReadWriteNeeded(func):
+    def wrapper(*args, **kwargs):
+        if current_user.role == 'Read/Write' or os.getenv('LOGIN_DISABLED') == 'True':
+            return func(*args, **kwargs)
+        else: return redirect('/')
```

Comment:
This works but is arguably confusing - the user doesn't get any sign the request has failed. Possibly returning a clear error message would be better, for example:
```
else: return "Not authorised", 401
```